tigerpath/majors_and_certificates/majors/GEO.json

```diff
@@ -57,7 +53,11 @@
           "explanation": "Three prerequisite requirements for entering the concentration (MAT 104 (or AP), COS 126, and GEO 202 or 203). Students should elect GEO 202 or 203 with an eye toward their likely track of study.\n\nWith permission of the DUS, a student can substitute GEO 102/103, a Geosciences Freshman Seminar, or certain Geosciences 300/400 level courses for GEO 202/203.",
           "course_list": [
             "GEO 202",
-            "GEO 203"
+            "GEO 203",
+            "GEO 102",

```

Comment:
https://geosciences.princeton.edu/sites/geosciences/files/media/information_and_departmental_plan_of_study-revised-02.pdf

"1b)	With permission of	the Director of Undergraduate Studies (DUS), a student can substitute GEO 102 or	GEO	103,	a Geosciences Freshman Seminar, or certain Geosciences	300/400	level	courses for GEO 202 or	GEO	203.	"

Reply:
Since it requires permission, I think it's better to leave it out. It doesn't seem like the "standard" track that people should take, and including it makes it seem like anyone can do it.

Reply:
Gotcha, fixed.